custom/icds_reports/ucr/data_sources/dashboard/postnatal_care_forms.json

```diff
@@ -290,6 +290,33 @@
           "type": "property_name",
           "property_name": "not_breastfeeding"
         }
+      },
+       {
+        "column_id": "unscheduled_visit",
+        "datatype": "small_integer",
+        "type": "expression",
+        "expression": {
+          "type": "property_path",
+          "property_path": ["form", "unscheduled_visit"]
+        }
+      },
+      {
+        "column_id": "late_visit",
+        "datatype": "small_integer",
+        "type": "expression",
+        "expression": {
+          "type": "property_path",
+          "property_path": ["form", "days_visit_late"]
+        }
+      },
+      {
+        "column_id": "next_visit",
```

Comment:
how are you planning to build these tables? given our current backlog, i'm skeptical it will be done anywhere near in time